int32 index
int32 mot1_vel_sps
int32 mot2_vel_sps